### PR TITLE
Miyazono research patch

### PR DESCRIPTION
**Pull request by miyazono**
Added some brief open problem statements and updated the research repo readme to create a nice research README.md

### CODE REVIEW

**Review by dignifiedquire — APPROVED** [review]

File: open-problems/brief-problem-statements.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+The most important problems/technologies relevant to Protocol Labs that will/should exist 5-10 years from now:
+
+- certifiable archives (.car)
```
Comment (daviddias) [inline open-problems/brief-problem-statements.md]: Shouldn't this one be on the ipfs/research?

**Review by daviddias — APPROVED** [review]
One question, other than that LGTM

File: open-problems/brief-problem-statements.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- anonymous, private connectivity
+	- distinction between ephemeral and permanent IDs
+- better congestion control
+- libp2p as a service (on your computer)
+- better DHTs
+	- lower latency, more secure
+- packet switching
+	- make libp2p packet oriented instead of stream oriented (currently streams on packets on streams on packets)
```
Comment (ghost) [inline open-problems/brief-problem-statements.md]: libp2p-as-a-service and packet switching aren't open research problems. We know exactly what to do, it just needs doing

**Comment by ghost** [comment]
LGTM 👍 Left one comment

File: open-problems/brief-problem-statements.md (hunk 1 above)
Comment (miyazono) [reply open-problems/brief-problem-statements.md]: No harm in adding it to the list in IPFS/research.  As it came up in the conversation, it was treated as part of libp2p (I think to put it in the lowest library that would use it), but I think you could make a good argument for it being in IPFS, IPLD, or probably even multiformats.

**Pull request merged**